warm_transfer_flask/call.py

```diff
@@ -1,12 +1,12 @@
-from twilio.rest import TwilioRestClient
+from twilio.rest import Client
 from os import environ as ENV
 
 
 def call_agent(agent_id, callback_url):
     account_sid = ENV['TWILIO_ACCOUNT_SID']
     auth_token = ENV['TWILIO_AUTH_TOKEN']
     my_number = ENV['TWILIO_NUMBER']
-    client = TwilioRestClient(account_sid, auth_token)
+    client = Client(account_sid, auth_token)
     call = client.calls.create(to='client:' + agent_id, from_=my_number,
```

Comment:
@acamino I totally agree with you, but notice that in this specific case, [both `to` and `from_` are required parameters](https://github.com/twilio/twilio-python/blob/next-gen/twilio/rest/api/v2010/account/call/__init__.py#L45).

If we change, it would be something like: 
```python
call = client.calls.create('client:' + agent_id, my_number,
```

Should I really proceed with this change anyway?